brax/io/html.py

```diff
@@ -176,7 +222,7 @@ def render(sys: brax.System, qps: List[brax.QP]) -> str:
   scene.add(hemiLight);
 
   const dirLight = new THREE.DirectionalLight(0xffffff);
-  dirLight.position.set(3, 10, 10);
+  dirLight.position.set(3, 5, 10);
```

Comment:
I see a few changes: this line, the autoRotate change below, and the camera distance change below - do you have a specific scene that needs these changes?  I would suggest rather than changing them for all scenes, instead either:

1) expose them as properties in the gui
2) find clever ways of automatically determining good defaults given the scene information
3) leaving the defaults as is for now if 1) or 2) is too tricky.  We can always add an issue for determining better defaults and solve it in a different PR.

Reply:
Those changes were only done to have a better shading on the terrain and validate the underlying geometry. I will revert those changes to default.